### PR TITLE
Goformatted the package and added goreportcard A+ badge.

### PR DESCRIPTION
**Pull request by nikmahes**
**Two Commits :**

1. goformatting the package to fix goreportcard issues
2. Updating README with goreportcard A+ badge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## goCron: A Golang Job Scheduling Package.
 [![GoDoc](https://godoc.org/github.com/golang/gddo?status.svg)](http://godoc.org/github.com/jasonlvhit/gocron)
 [![Stories in Ready](https://badge.waffle.io/jasonlvhit/gocron.png?label=ready&title=Ready)](https://waffle.io/jasonlvhit/gocron)
+[![Go Report Card](https://goreportcard.com/badge/github.com/nikmahes/gocron)](https://goreportcard.com/report/github.com/nikmahes/gocron)
```
Comment (Depado) [inline README.md]: I think the badge is pointing to your fork and not the original repository

File: gocron_test.go [hunk 2]
```diff
@@ -2,9 +2,9 @@
 package gocron
 
 import (
+	"fmt"
```
Comment (MovieStoreGuy) [inline gocron_test.go]: Is this change really needed?
